ansible/roles/pmm2-images/tasks/main.yml

```diff
@@ -107,6 +107,7 @@
         name:
           - percona-grafana
           - percona-prometheus
+          - percona-VictoriaMetrics
```

Comment:
in line 3 of this file:
`# All new plays should be added into pmm-update playbook. It is invoked after pmm2 playbook.`

Reply:
Got it! See https://github.com/percona/pmm-update/pull/96 .

Reply:
Abandoning this PR because of https://github.com/percona/pmm-update/pull/96

Reply:
Got it! See https://github.com/percona/pmm-update/pull/96

Reply:
After a lot of discussions and attempts to fix things, we decided that new packages should be added in both places – there and in pmm-update

Reply:
`percona-victoriametrics`, lowercase?

Reply:
done